robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -52,7 +51,7 @@ public MenuItem add(int groupId, int itemId, int order, CharSequence title) {
 
   @Override
   public MenuItem add(int groupId, int itemId, int order, int titleRes) {
-    return add(groupId, itemId, order, context.getResources().getString(titleRes));
+    return add(groupId, itemId, order, RuntimeEnvironment.application.getResources().getString(titleRes));
```

Comment:
1. No, sorry.
2. Sorry, I misspoke. Referring to this: https://github.com/robolectric/robolectric/blob/master/robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenuItem.java#L102